src/Symfony/Component/HttpKernel/KernelEvents.php

```diff
@@ -29,27 +29,6 @@ final class KernelEvents
      */
     const REQUEST = 'kernel.request';
 
-    /**
-     * The EXCEPTION event occurs when an uncaught exception appears.
-     *
-     * This event allows you to create a response for a thrown exception or
-     * to modify the thrown exception.
-     *
-     * @Event("Symfony\Component\HttpKernel\Event\ExceptionEvent")
-     */
-    const EXCEPTION = 'kernel.exception';
```

Comment:
I'd suggest keeping this one here as it can very well happen before all the other ones.

Reply:
I'm not conviced by this suggestion. Altought the exception event could occurs whenever in the lifecycle, it may not occurs at all which make it different from all the others. That's why to me it should be either on the first or the last position. What do you think ?

Reply:
I think we should move things if there's a clear reason to do it. Doesn't seem to be the case here.

Reply:
I agree with @nicolas-grekas. Now, I'm not convinced that we should spend time on such changes. So, better change as little as possible.